### DISCUSSION TITLE
https://maxkim-j.github.io/posts/suspense-argibraic-effect

### DISCUSSION THREAD

**Discussion by MaxKim-J** [MaxKim-J/maxkim-j.github.io#78] [Announcements]
#          🪄 Suspense for Data Fetching의 작동 원리와 컨셉 (feat.대수적 효과)      

https://maxkim-j.github.io/posts/suspense-argibraic-effect

**Comment by MaxKim-J** [comment]
<p>Suspense 의 동작원리를 보면서 throw 를 이용해서 상위 콘텍스트에 promise를 전달하는거는 진짜 발상의 전환이네.. 정도로만 생각했었는데, 이를 뒷받침하는 많은 로직들이 있었네요..! <br>100% 이해하지는 못하고 가볍게 훑어만 보았는데, 이 글을 다 이해할 수 있도록 더 학습해야겠습니다!</p><p>좋은 글 공유해주셔서 감사합니다!</p>

>_This comment was made by Disqus. 2021-11-03 08:40:10 Donghee Kim(bigsaigon)_

**Comment by MaxKim-J** [comment]
<p>좋은 글 감사합니다. 공부가 되었습니다.</p>

>_This comment was made by Disqus. 2021-11-10 09:26:12 김현진(disqus_vC43u1fNEH)_

**Comment by moonheekim0118** [comment]
좋은 글 감사합니다 :)

**Comment by kohj1018** [comment]
좋은 글 감사합니다! 많은 공부가 되었습니다 :)

**Comment by Tchaikovsky1114** [comment]
너무 좋은 글이었습니다..!! 겸손해지는 시간이었습니다.. 감사합니다!!

**Comment by jeffbukk00** [comment]
좋은 글 감사합니다!
